### PR TITLE
History_metadata

### PR DESCRIPTION
**Pull request by woolfg**
schema changed in https://app.planetscale.com/services-openpodcast/openpodcast/deploy-requests/22 deployed

### CODE REVIEW

File: src/db/SpotifyRepository.ts [hunk 1]
```diff
@@ -33,8 +33,30 @@ class SpotifyRepository {
...
+            spm_streams,
+            spm_listeners,
+            spm_followers ) VALUES
+            (?,?,?,?,?,?,?)`
+
+        const today = new Date()
+
+        return await this.pool.query(replaceStmtHistory, [
             accountId,
+            `${today.toLocaleDateString('en-CA')}`, //format to YYYY-mm-dd
```
Comment (mre) [inline src/db/SpotifyRepository.ts:59]: why `en-CA` and not `en-US`? 🤔

**Comment by mre** [comment]
Is this still a WIP or don't we need any API endpoints for this change?

**Comment by woolfg** [comment]
> Is this still a WIP or don't we need any API endpoints for this change?

no need for an api, everything done in house ;)

File: src/db/SpotifyRepository.ts (hunk 1 above)
Comment (woolfg) [reply src/db/SpotifyRepository.ts:59]: because US returns "11/28/2022" format...

**Review by mre — APPROVED** [review]

**Pull request merged**